uint32 from_row
uint32 from_col
---
bool success